apps/server/src/modules/board/controller/mapper/content-element-response.factory.ts

```diff
@@ -3,11 +3,12 @@ import { AnyBoardDo, FileElement, RichTextElement } from '@shared/domain/domaino
 import {
 	AnyContentElementResponse,
 	FileElementResponse,
+	RichTextElementResponse,
 	isFileElementResponse,
 	isRichTextElementResponse,
-	RichTextElementResponse,
 } from '../dto';
 import { BaseResponseMapper } from './base-mapper.interface';
+import { CollaborativeTextEditorElementResponseMapper } from './collaborative-text-editor-element-response.mapper';
```

Comment:
Again please combine it with the `Response` import from above.

Reply:
How could they even be combined?